requirements.txt

```diff
@@ -1,14 +1,14 @@
-Django<2.0
+Django==1.11.22
```

Comment:
> What about **no**?

We shouldn't add such precise dependencies in `requirements.txt`. Minor version updates should be allowed. Could you tell me why the issue I showed happens?